Epsilon.Canvas.Abstractions/Service/IFileHttpService.cs

```diff
@@ -0,0 +1,7 @@
+namespace Epsilon.Canvas.Abstractions.Service;
+
+public interface IFileHttpService
+{
+    Task<byte[]> GetFileByteArray(string url);
```

Comment:
Return IEnumerable<byte> and also should be specific to Canvas, so file id maybe?

Reply:
changed